go-sdk/sdk/centralized-configuration/centralized_configuration_get_test.go

```diff
@@ -8,7 +8,6 @@ import (
 
 	"github.com/konstellation-io/kai-sdk/go-sdk/mocks"
 	centralizedconfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```

Comment:
```suggestion
	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```
Ditto, replace with camelCase